bindings/time-profiler.cc

```diff
@@ -67,12 +67,31 @@ NAN_METHOD(StartProfiling) {
 NAN_METHOD(StopProfiling) {
   Local<String> name = info[0].As<String>();
   CpuProfile* profile =
-    info.GetIsolate()->GetCpuProfiler()->StopProfiling(name);
+      info.GetIsolate()->GetCpuProfiler()->StopProfiling(name);
   Local<Value> translated_profile = TranslateTimeProfile(profile);
   profile->Delete();
   info.GetReturnValue().Set(translated_profile);
 }
 
+void free_buffer_callback(char* data, void* buf) {
```

Comment:
You can just use a lambda, no need to have a separate function. (Also I don't think this name here is per style guide).